test/unit/app/tools/test_evaluation.py

```diff
@@ -45,7 +45,7 @@ def setUp(self):
         self.job.history = History()
         self.job.history.id = 42
         self.job.parameters = [JobParameter(name="thresh", value="4")]
-        self.evaluator = ToolEvaluator(self.app, self.tool, self.job, self.test_directory)
+        self.evaluator = ToolEvaluator(self.app, self.tool, self.job, self.test_directory)  # type: ignore[arg-type]
```

Comment:
I have a patch (that touches other files) to actually fix this error, let me know if you'd like me to push it to your branch.

Reply:
Yes, please!